exonum/src/sandbox/consensus.rs

```diff
@@ -3074,6 +3076,90 @@ fn handle_tx_has_full_propose() {
     sandbox.add_time(Duration::from_millis(0));
 }
 
+/// HANDLE block response
+
+/// - should process block even if tx in pool
```

Comment:
I would extend the description a little bit. Additionally, do we need a doc-comments here?